pkg/local_object_storage/metabase/lock_test.go

```diff
@@ -169,6 +169,27 @@ func TestDB_Lock(t *testing.T) {
 	})
 }
 
+func TestDB_Lock_Expired(t *testing.T) {
+	es := &epochState{e: 123}
+
+	db := newDB(t, meta.WithEpochState(es))
+
+	// put an object
+	addr := putWithExpiration(t, db, object.TypeRegular, 124)
+
+	// expire the obj
+	es.e = 125
+	_, err := metaGet(db, addr, false)
+	require.ErrorIs(t, err, meta.ErrObjectIsExpired)
+
+	// lock the obj
+	require.NoError(t, db.Lock(addr.Container(), oidtest.ID(), []oid.ID{addr.Object()}))
```

Comment:
Can we _lock_ an already expired object or is it only for tests here?

Reply:
> Can we lock an already expired object

yes, and i do not see any problems with that, you? in practice i think that is almost impossible if GC is configured to be relatively fast

Reply:
Again, not for this PR.